samples/outlook/preview-apis-signature/work-with-client-signatures.yaml

```
@@ -0,0 +1,115 @@
+id: outlook-signature-work-with-client-signatures
+name: Work with Client Signatures
+description: 'Check if the client signature is enabled, disable the client signature, get the compose type, and set a signature.'
+host: OUTLOOK
+api_set:
+    Mailbox: '1.1'
+script:
+    content: |
+        $("#isClientSignatureEnabled").click(isClientSignatureEnabled);
+        $("#disableClientSignature").click(disableClientSignature);
+        $("#getComposeType").click(getComposeType);
+        $("#setSignature").click(setSignature);
+
+        function isClientSignatureEnabled() {
+          // Check if the client signature is currently enabled
```

We usually end code comments with a period e.g.
```suggestion
          // Check if the client signature is currently enabled.
```